package.json

```diff
@@ -18,28 +18,50 @@
     "scripts": {
         "dev": "rollup -m -c",
         "build": "rimraf build && rollup -m -c --environment NODE_ENV:production",
-        "test": "mocha"
+        "postbuild": "echo '{\"type\": \"commonjs\"}'> build/cjs/package.json",
+        "test": "npm run test:node && npm run test:browser",
+        "test:node": "NODE_OPTIONS='--experimental-vm-modules --no-warnings' jest --coverage",
+        "test:browser": "NODE_OPTIONS='--experimental-vm-modules --no-warnings' jest --coverage --testEnvironment=jsdom"
```

Comment:
Although JSDOM looks like a browser, it just emulates a browser-like environment. Which might lead to inconsistencies compared to testing this in an actual browser.

Reply:
That's true, but we are now running tests against JSDom and Node, which is an improvement over only a browser. I can see if we can bring in puppeteer and headless chrome tho 👍

Reply:
Probably this could be introduced in a later PR, if you like I can take a look at this.

Reply:
We are always open for contributions. If you are happy to take a stab at it, happy to review a PR!

Reply:
Sure, if you have no issue with it, I will try to open up a couple of PRs. Don't hold up this PR for it though :)

Reply:
I assume jsdom uses node's atob, and we want puppeteer to use chrome's atob?

Reply:
> I assume jsdom uses node's atob

Could be, I am not entirely sure. Either way, we should test on real browsers eventually so it doesn't make a difference now.